assignment 5/day 6/index.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const files = [
+  "./files/mongodb.txt",
+  "./files/expressjs.txt",
+  "./files/angularjs.txt",
+  "./files/nodejs.txt",
+  "./files/fullStack.txt"
+];
+
+function readFile(file, index) {
+  try {
+    fs.readFile(file, 'utf8', (err, data) => {
+      if (err) {
+        throw err;
+      }
+      console.log(data);
+      index++;
+      if (index < files.length) {
+        readFile(files[index],
+          index);
+      }
+    });
+  } catch (err) {
+    readFile(file, index);
+  }
+}
+
+readFile(files[0], 0);
```

Comment:
The error handling should be handled here at line 28. In this you have skipped the file in which Error has occurred. While you should notify the error gracefully instead of getting thrown.